e2eAIOK/deltatuner/deltatuner/search/utils.py

```diff
@@ -0,0 +1,137 @@
+import sys
+import os
```

Comment:
@tianyil1 the less duplicated/modified files from de-nas, the better. Please keep it minimum.

Reply:
Have removed the unnecessary part used for the previous DE-NAS.